tests/db_functions/test_cast.py

```diff
@@ -19,6 +21,12 @@ def test_cast_from_field(self):
         numbers = Author.objects.annotate(cast_string=Cast('age', models.CharField(max_length=255)),)
         self.assertEqual(numbers.get().cast_string, '1')
 
+    @mock.patch('sys.stderr')
+    @skipIf(connection.vendor == 'sqlite', "SQLite doesn't support Cast with precision.")
```

Comment:
I guess it could be better as a database feature in case any third-party databases also have that limitation.